Classes/Domain/Extractor/ExifExtractor.php

```diff
@@ -151,9 +151,13 @@ class ExifExtractor extends AbstractExtractor
     public function extractMetaData(FlowResource $resource, MetaDataCollection $metaDataCollection)
     {
         try {
-            $exifData = @exif_read_data($resource->createTemporaryLocalCopy(), 'EXIF');
+            $exifData = @\exif_read_data($resource->createTemporaryLocalCopy(), 'EXIF');
         } catch (FlowResourceException $exception) {
-            throw new ExtractorException('Could not extract EXIF data from ' . $resource->getFilename(), 1484059228, $exception);
+            throw new ExtractorException(
+                'Could not extract EXIF data from ' . $resource->getFilename(),
```

Comment:
Personally I don't like when things like exceptions and log messages dominate the code, but ...